### PR TITLE
Serving notifications

### PR DESCRIPTION
**Pull request by maxov**
@stkem 
@andrewconner 
@leogrim 
@ryanpbrewster 

Still a WIP

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/notify/info/StandardNotificationInfo.scala [hunk 1]
```diff
@@ -1,7 +1,8 @@
 package com.keepit.notify.info
 
-import com.keepit.common.db.Id
+import com.keepit.common.db.{ ExternalId, Id }
```
Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/notify/info/StandardNotificationInfo.scala]: The added import of `ExternalId` seems unused in the diff. Am I missing something?

**Comment by stkem** [comment]
code looks good to me on first read through. Do we have test coverage on the modified websocket routes here now?

**Comment by maxov** [comment]
@stkem No, it's possible to add some tests but it would be quite difficult because of all of the moving parts.. I'll definitely try to do that when the logic is more finished. For now I'm just testing it locally, running with a dev server.

File: kifi-backend/modules/common/app/com/keepit/notify/info/StandardNotificationInfo.scala (hunk 1 above)
Comment (maxov) [inline kifi-backend/modules/common/app/com/keepit/notify/info/StandardNotificationInfo.scala]: thanks. I removed the bad imports

**Comment by maxov** [comment]
@stkem I added tests, this look good to merge?

**Comment by stkem** [comment]
:+1:

**Pull request merged**